tests/test_pipreqs.py

```diff
@@ -9,18 +9,27 @@
 """
 
 from io import StringIO
-from unittest.mock import patch
```

Comment:
why is this import being deleted? and what was it used for

Reply:
cross-contamination, just solved it.